java-squid/src/test/java/org/sonar/java/cfg/CFGTest.java

```diff
@@ -380,8 +373,7 @@ public void if_then() {
     final CFG cfg = buildCFG("void fun() {if(a) { foo(); } }");
     final CFGChecker cfgChecker = checker(
       block(
-        element(Tree.Kind.IDENTIFIER, "a")
-        ).terminator(Tree.Kind.IF_STATEMENT).successors(0, 1),
+        element(Tree.Kind.IDENTIFIER, "a")).terminator(Tree.Kind.IF_STATEMENT).successors(0, 1),
```

Comment:
This class is heavily modified just because of formatting, this make the review unecessarily hard.